src/content-helper/editor-sidebar/smart-linking/smart-linking.scss

```
@@ -161,6 +166,6 @@
 	}
 
 	.wp-parsely-smart-linking-hint {
-		margin: 0;
+		margin-top: var(--grid-unit-20, to_rem(16px));
```

```suggestion
		margin-top: var(--grid-unit-20);
```
Going forward, no need to add a fallback value for variables we're declaring ourselves.